.internal-ci/helm/consensus-node/values.yaml

```
@@ -102,35 +108,34 @@ node:
     - -c
     - |
       set -e
-      if [ ! -f /ledger/data.mdb ]; then
-        if [[ -n "${INITIAL_KEYS_SEED}" ]]; then
+      if [ ! -f /ledger/data.mdb ]
+      then
+        if [[ -n "${INITIAL_KEYS_SEED}" ]]
+        then
           echo "INITIAL_KEYS_SEED found - populating origin data"
           export INITIALIZE_LEDGER="true"
           /usr/local/bin/generate_origin_data.sh
           cp /tmp/sample_data/ledger/data.mdb /ledger
-        elif [ -f /var/lib/mobilecoin/origin_data/data.mdb ]; then
+          exit 0
+        fi
+
+        echo "Get ledger from S3 - ledger-from-archive"
+        # Check to see if archive exists
+        curl -I -f "${TX_SOURCE_URL}/00/00/00/00/00/00/00/0000000000000000.pb"
+        /usr/bin/ledger-from-archive --ledger-db /ledger --tx-source-url "${TX_SOURCE_URL}"
+        exit 0
```

This unconditional exit doesn't look right.
```suggestion
        if curl -I -f "${TX_SOURCE_URL}/00/00/00/00/00/00/00/0000000000000000.pb"
        then
          /usr/bin/ledger-from-archive --ledger-db /ledger --tx-source-url "${TX_SOURCE_URL}"
          exit 0
        fi
```